packages/p2p/src/components/orders/chat/chat-messages.jsx

```diff
@@ -23,13 +23,6 @@ const ChatMessages = observer(() => {
         }
     };
 
-    React.useEffect(() => {
-        if (sendbird_store.chat_messages.length > 0 && scroll_ref.current) {
-            // Scroll all the way to the bottom of the container.
-            scroll_ref.current.scrollTop = scroll_ref.current.scrollHeight;
-        }
-    }, [sendbird_store.chat_messages.length]); // eslint-disable-line react-hooks/exhaustive-deps
-
```

Comment:
this is the main issue, because the chat_messages are always updating, even when we get the old messages, this will scroll down to the bottom of the chat. I believe this was used to help when you send a message, the chat needs to scroll down automatically